docs/schema.md

```diff
@@ -1196,6 +1196,15 @@ Filter by 'is_marketplace' field of system profile
 
 Filter by 'operating_system' field of system profile
 
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>spf_ansible</strong></td>
+<td valign="top"><a href="#filteransible">FilterAnsible</a></td>
+<td>
+
+Filter by 'ansible' field of system profile
```

Comment:
The docs here are missing descriptions about the use of the sub-fields of spf_ansible because the type is not defined in the graphql schema, as noted in another comment.